user/models.py

```diff
@@ -1,5 +1,52 @@
-from django.contrib.auth.models import AbstractUser
+from django.contrib.auth.models import (
+    AbstractUser,
+    UserManager as DjangoUserManager
+)
+from django.db import models
+from django.utils.translation import gettext as _
+
+
+class UserManager(DjangoUserManager):
+    """Define a model manager for User model with no username field."""
```

Comment:
The docstring mentions that the User model has no username field, which is accurate. Ensure that this is reflected consistently across your documentation and code comments.